src/components/EditClass.js

```diff
@@ -0,0 +1,141 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+import 'airbnb-js-shims' // to allow jest to understand Object.entries for parsing the character objects
+
+import { Button, Modal } from 'react-bootstrap'
+
+/* global fetch */
+
+const SUCCESS_STYLE = 'success'
+const DEFAULT_STYLE = 'default'
+
+export default class EditClass extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      showModal: false,
+      klasses: {},
```

Comment:
ugh, the mixture of `klass` and `class` is pretty unmaintainable. I see you using `klass` instead of `class`, which makes sense, but then you're not consistent with `klasses` versus `classes` and `EditClass` versus `EditKlass`. I would suggest that you get rid of `klass` everywhere and simply use `className` or `characterClass` everywhere.